README.md

```diff
@@ -1,28 +1,32 @@
 # Go-SSB Room
 
-This repository contains code for a [Secure-Scuttlebutt Room v2](https://github.com/ssb-ngi-pointer/rooms2/) server written in Go.
+This repository contains code for a [Secure Scuttlebutt](https://ssb.nz) [Room (v1+v2) server](https://github.com/ssb-ngi-pointer/rooms2), written in Go.
 
-It not only includes the secret-handshake+boxstream setup and muxrpc handlers for tunneling connections but also a fully embedded http/html interface for administering the room.
+It includes:
+* a secret-handshake+boxstream setup
+* muxrpc handlers for tunneling connections
+* a fully embedded server & frontend, for administering the room
```

Comment:
```suggestion
* a fully embedded HTTP server & HTML frontend, for administering the room
```

Reply:
in general in the intro: i was trying to keep it kinda minimal & tidy as it's the first thing people see :) guessing the majority of people won't care about the details, so much as having a nice room to deploy.

if the details are important for you in this part though, i'll just add them :)

Reply:
I think we just spell out what kind of frontend is to be expected and disambiguated that there is a muxrpc and a http server here but I don't think it's necessary.